src/enigma2/ChannelGroups.h

```diff
@@ -49,6 +49,7 @@ namespace enigma2
     void ClearChannelGroups();
     std::vector<std::shared_ptr<enigma2::data::ChannelGroup>>& GetChannelGroupsList();
     bool LoadChannelGroups();
+    std::vector<std::shared_ptr<enigma2::data::ChannelGroup>>& GetExtraDataChannelGroupsList() { return m_extraDataChannelGroups; };
```

Comment:
Can this method be const?

Reply:
No, as it returns a vector.

Reply:
Huh? Cannot see how this should be related.

Reply:
The returned vector reference could even be const, not only the method, if  I read the code correctly.

Reply:
You're right, I didn't make the returned reference const at the same time!

Reply:
Ooops